### ISSUE TITLE
in cpplint a compound literal in return statement gives readability/brace warning

### ISSUE THREAD

**Issue by alpo** [google/styleguide#311]
## Actions:

Check with cpplint a source code with 2 fragments:

1:
```c
return (SomeType){.some_field = some_value};
```

2:
```c
SomeType x = (SomeType){.some_field = some_value};
return x;
```

## Expected result: 

Cpplint will not emit the message "You don't need a ; after a }  [readability/braces]" in both cases

## Actual result:

Cpplint emits the warning in the 1st case.

## Suggestion:

Add "return" statement to the line prefixes blocking this warning for compound literals.

**Comment by alpo** [comment]
https://github.com/google/styleguide/pull/312

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see https://github.com/google/styleguide/pull/837). please migrate to https://github.com/cpplint/cpplint now.
